xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-sources/xwiki-platform-localization-source-wiki/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------------------------
+# See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
+
+###################
+# Required rights
+###################
+
+localization.requiredrights.translationDocument.global=A translation document with global scope requires programming \
+  rights.
+localization.requiredrights.translationDocument.user=This wiki instance enforces that a translation document with user \
+  scope requires script rights.
+localization.requiredrights.translationDocument.wiki=A translation document with wiki scope requires administrator \
```

Comment:
Admin rights can be at different levels (e.g., also at space level). To make this clear, I think we should write "wiki administration rights". Same for the message for the content.